### PR TITLE
Try and address the first round of WG comments on the charter update

### PR DESCRIPTION
**Pull request by larseggert**

### CODE REVIEW

File: charter/charter.md [hunk 1]
```diff
@@ -23,29 +23,27 @@ to carefully consider its impact on the diverse set of applications
...
 The third area of work is the specification of new extensions to QUIC.
-Extensions intended for Standards Track need to have general
-applicability to multiple application protocols. The WG may also
-decide to publish extensions as Informational or Experimental
-documents, e.g., to allow vendors to publicly document deployed
-proprietary extensions or to enable wider experimentation with new
-protocol features.
+The WG will primarily focus on extensions to the QUIC transport layer,
+i.e., extensions to QUIC that have broad applicability to multiple
+application protocols. The WG may also allow publish specifications to
```
Comment (LPardue) [inline charter/charter.md:33]: Allow publish?

Comment (larseggert) [reply charter/charter.md:33]: Picky, picky :-) Yeah, the "allow" should not be there.

Comment (larseggert) [reply charter/charter.md:33]:
```suggestion
application protocols. The WG may also publish specifications to
```

**Review by LPardue — APPROVED** [review]

**Review by DavidSchinazi — APPROVED** [review]

**Review by martinthomson — APPROVED** [review]
Also, the charter is good.

**Comment by SpencerDawkins** [comment]
I THINK I'm reading this as the QUIC working group **requesting** groups that realize that their applications require QUIC extensions to consult with the QUIC working group, and seek review. Is that the intention?

I'd expect that to be stronger, simply because (based on experiences with protocols like SIP) popular protocols tend to collect applications from people who don't understand the underlying protocol as well as the people who are responsible for the underlying protocol. If you can say "but you can accomplish the same thing by using QUIC as it is now", sooner rather than later, that would probably make everyone's lives simpler. 

In addition to just being nice to other people, I'd kind of expect that review will happen eventually for any QUIC extension that makes it to Publication Requested status in its own working group and goes for IETF Last Call, whether the review happens because a TSVART reviewer is assigned, or because some QUIC expert saw the Last Call announcement and became curious. But that's about as late as Late Surprises ever get. 

So, maybe that could say something like "are encouraged to consult with the QUIC WG and obtain early review of proposals, thereby avoiding late surprises"? 

Do the right thing, of course.

File: charter/charter.md [hunk 2]
```diff
@@ -23,29 +23,27 @@ to carefully consider its impact on the diverse set of applications
...
-of proposals. This is intended to reduce the possibility of duplicate
-work and/or conflicts with other extensions.
+use the QUIC transport layer need not be specified in the QUIC WG,
+although they can. The QUIC WG will collaborate with other groups that
+define such application protocols that intend to use QUIC. New
+mappings might require QUIC extensions and it may be efficient to
+define these alongside the mapping specifications. Groups that define
+application protocols using QUIC, or extensions to QUIC in support of
+those protocols, are requested to consult with the QUIC WG and seek
+review of proposals. This is intended to reduce the possibility of
```
Comment (larseggert) [inline charter/charter.md:45]: Trying to capture the suggestion from @SpencerDawkins:

```suggestion
define these alongside the mapping specifications. Groups that define
application protocols using QUIC, or extensions to QUIC in support of
those protocols, are strongly requested to consult with the QUIC WG and seek
early and ongoing review of and collaboration on proposals. This is intended to reduce the possibility of
```

Comment (SpencerDawkins) [reply charter/charter.md:45]: @larseggert captured my suggestion and improved the heck out of it. Thank you for that!

**Comment by seanturner** [comment]
How does the 1st area of work and 3rd area of work differ? Both mention extensions.

1st area:

    Maintenance and evolution of the specifications of QUIC extensions.

3rd area:

    The third area of work is the specification of new extensions to QUIC.

**Comment by larseggert** [comment]
See email.

**Pull request merged**